manifests/charts/cloudcore/templates/rbac_cloudcore.yaml

```diff
@@ -14,7 +14,7 @@ rules:
   resources: ["namespaces"]
   verbs: ["get", "create", "list", "watch"]
 - apiGroups: [""]
-  resources: ["nodes", "pods/status"]
+  resources: ["nodes", "nodes/status", "pods/status"]
```

Comment:
The CI haven't reported the error? cc @gy95

Reply:
keadm e2e contains `Create a pod and check the pod is coming up correctly`
https://github.com/kubeedge/kubeedge/blob/master/tests/e2e/keadm/keadm_test.go
maybe not cover
how to trigger this? where do we need patch methods for nodes/status?

Reply:
https://github.com/kubeedge/kubeedge/blob/340a251b44d5d8e28061758e86345f81190882f3/cloud/pkg/edgecontroller/controller/upstream.go#L968 in new edged, which update the node status

Reply:
```
Sep 26 14:52:43 wackxu-dev edgecore[3611914]: E0926 14:52:43.030707 3611914 kubelet_node_status.go:400] "Error updating node status, will retry" err="failed to patch status \"{\\\"status\\\":{\\\"$setElementOrder/conditions\\\":[{\\\"type\\\":\\\"MemoryPressure\\\"},{\\\"type\\\":\\\"DiskPressure\\\"},{\\\"type\\\":\\\"PIDPressure\\\"},{\\\"type\\\":\\\"Ready\\\"}],\\\"conditions\\\":[{\\\"lastHeartbeatTime\\\":\\\"2022-09-26T06:52:43Z\\\",\\\"type\\\":\\\"MemoryPressure\\\"},{\\\"lastHeartbeatTime\\\":\\\"2022-09-26T06:52:43Z\\\",\\\"type\\\":\\\"DiskPressure\\\"},{\\\"lastHeartbeatTime\\\":\\\"2022-09-26T06:52:43Z\\\",\\\"type\\\":\\\"PIDPressure\\\"},{\\\"lastHeartbeatTime\\\":\\\"2022-09-26T06:52:43Z\\\",\\\"type\\\":\\\"Ready\\\"}]}}\" for node \"e2e-test\": nodes \"e2e-test\" is forbidden: User \"system:serviceaccount:kubeedge:cloudcore\" cannot patch resource \"nodes/status\" in API group \"\" at the cluster scope"

Reply:
> keadm e2e contains `Create a pod and check the pod is coming up correctly` https://github.com/kubeedge/kubeedge/blob/master/tests/e2e/keadm/keadm_test.go maybe not cover how to trigger this? where do we need patch methods for nodes/status?

IIRC we already have a test in keadm_e2e which builds the container from source code? If the rule `nodes/status` missed, the node status can't be ready?

Reply:
in my environment that use keadm install , the node status is always not ready

```
root@open-source-dev:~/test-edge-pod# kubectl get node
NAME        STATUS     ROLES        AGE     VERSION
127.0.0.1   Ready      <none>       3h48m   v1.22.6-dirty
e2e-test    NotReady   agent,edge   14m     v1.22.6-kubeedge-v1.11.0-beta.0.211+347278db67ea3d-dirty

```

Reply:
https://github.com/kubeedge/kubeedge/blob/340a251b44d5d8e28061758e86345f81190882f3/tests/e2e/scripts/keadm_e2e.sh#L75-L79
maybe this check is not correct, just like https://github.com/kubeedge/kubeedge/pull/4231

Reply:
> maybe this check is not correct, just like https://github.com/kubeedge/kubeedge/pull/4231

If not correct, the test case should not passed :)

Reply:
yes, by test in https://github.com/kubeedge/kubeedge/pull/4235, it indeed is

Reply:
can fix like https://github.com/kubeedge/kubeedge/pull/4235,
`grep -q Ready` -> `grep -q -w Ready`

Reply:
Besides the node `notready` status, the app seems have no node-selector? Or how the app can run now.